src/ashley/management/commands/sync_group_permissions.py

```diff
@@ -0,0 +1,102 @@
+"""
+This module provides a management command `sync_group_permissions`
+to help ashley operators maintaining a consistent set of forum permission
+across all LTI related groups.
+"""
+
+from django.core.management.base import BaseCommand
+from machina.apps.forum_permission.shortcuts import assign_perm, remove_perm
+from machina.core.db.models import get_model
+
+from ashley.defaults import DEFAULT_FORUM_ROLES_PERMISSIONS
+
+Forum = get_model("forum", "Forum")  # pylint: disable=C0103
+GroupForumPermission = get_model(
+    "forum_permission", "GroupForumPermission"
+)  # pylint: disable=C0103
+LTIContext = get_model("ashley", "LTIContext")  # pylint: disable=C0103
+
+
+class Command(BaseCommand):
+    """
+    Implementation of the sync_group_permission Command.
+    """
+
+    help = (
+        "Update group permissions in database with permissions defined "
+        "in ASHLEY_DEFAULT_FORUM_ROLES_PERMISSIONS. By default, only missing "
+        "permissions are added and no permission is removed."
```

Comment:
I would say the default should be to remove the permissions that were removed as it is safer. Adding permissions should be done voluntarily?

Reply:
Interresting, I thought the exact inverse :grin: 

Imagine, you manually granted extra permissions to a specific group on a forum (i.e. studends can upload attachment on a specific course forum only). If you remove permissions by default, it'll break these specific permissions.
With this case in mind, I thought that the default behavior of «add-only missing permissions» was safer.

But in the end, it seems that none of these operations are safe !